pyproject.toml

```diff
@@ -60,7 +60,7 @@ build-backend = "poetry_dynamic_versioning.backend"
 python = "^3.8"
 xmltodict = "^0.13.0"
 colorlog = "^6.7.0"
-kubernetes = "^30.0.0"
+kubernetes = "^29.0.0"
```

Comment:
why is this change needed?

Reply:
@rnetser OCP 4.16 is not based off kubernetes 1.30, it is based off kubernetes 1.29. 4.16 branch of wrapper should not be using kubernetes client 30.*.*, it should be using 29.0.0 instead. Any breaking changes of kubernetes clients are not ported to not related branches. This creates a potential problem for wrapper and it's consumer. We saw this while supporting 4.15.